socorro/processor/mozilla_processor_2015.py

```diff
@@ -1,6 +1,8 @@
-from ujson import dumps
+import ujson
```

Comment:
I can't put my finger on it but something tells me we should avoid optimization tricks like this. 
The regular standard python json module is good enough 99% of the cases. We use ujson in some parts where shitloads of JSON encoding and decoding is going on. 
So although I don't have a direct concrete example where it breaks; my instincts are at least asking me to raise a concern.